doofinder-for-woocommerce/includes/class-update-on-save.php

```diff
@@ -18,10 +18,9 @@ public static function init()
         //Force Update on save
         add_action('wp_ajax_doofinder_force_update_on_save', function () {
             do_action("doofinder_update_on_save");
-            wp_send_json([
-                'success' => true
-            ]);
+            wp_send_json_success();
             exit;
```

Comment:
In this case, exit shouldn't be needed since internally wp_send_json_success calls wp_send_jason and this one has a wp_die and die (https://developer.wordpress.org/reference/functions/wp_send_json/). But this is a minor detail. Approved, well done!! :clap: